sites/example-project/src/components/ui/Sidebar.svelte

```diff
@@ -92,6 +92,13 @@ aside.sidebar {
     border-right: 1px solid var(--grey-300);
 }
 
+.project-title{
+		font-size: 1.2em;
+		font-weight: bold;
+		color: var(--grey-800);
+		margin: 0 0 0.3em 0; 
+		padding-top: 0.3em;
+	}
```

Comment:
Hi, the indentation seems slightly off. There needs to be a small space after `.project-title`. Also, I am nor sure if this decision is intentional, but it's better to stick with `rem` units as `em`'s are relative unit and very delicate to work with on a group setting. But if this was intentional and everyone understands, then cool.

Reply:
We do have a more general need to standardize our approach on UI variables / UI design, but for the moment we're predominantly using `em` units.